.github/workflows/gitops-workflow.yaml

```
@@ -0,0 +1,106 @@
+name: GitOps Workflow
```

Forgive me being a little picky here - we can remove the word "Workflow" in the name here and the file name, as it's already undoubtedly a workflow.

This is like setting my nickname to "CAI YUWANG The Human", but everybody surely knows I'm a human.
```suggestion
name: GitOps
```